### DISCUSSION TITLE
Cannot access raspap-webgui once webmin installed

### DISCUSSION THREAD

**Discussion by bmaire28** [RaspAP/raspap-webgui#1488] [Q&A] [answered]
Hi,

I've installed raspap on raspberry 3b after a fresh install of bullseyes and using the quick installation.
It was working well.

I've modified the ssid and passwords and ip range.
Starting adding static ip records.

Then I've installed webmin, squid and squiGuard.

And from then, I cannot connect to the webgui anymore : getting "access denied" page if trying through http or connection refused through https.

I've read all topics on this site and other site but it seems no one had this error.

Squid config :
`# Fichier de config pour squid

# reseau local
acl netMaire src 10.3.141.0/24  # Local Wifi
http_access allow netMaire

# Configuration par defaut
# reseau non route
acl localnet src 0.0.0.1-0.255.255.255	# RFC 1122 "this" network (LAN)
acl localnet src 10.0.0.0/8		# RFC 1918 local private network (LAN)
acl localnet src 100.64.0.0/10		# RFC 6598 shared address space (CGN)
acl localnet src 169.254.0.0/16 	# RFC 3927 link-local (directly plugged) machines
acl localnet src 172.16.0.0/12		# RFC 1918 local private network (LAN)
acl localnet src 192.168.0.0/16		# RFC 1918 local private network (LAN)
acl localnet src fc00::/7       	# RFC 4193 local private network range
acl localnet src fe80::/10      	# RFC 4291 link-local (directly plugged) machines
# ports connus
acl SSL_ports port 443
acl Safe_ports port 80		# http
acl Safe_ports port 21		# ftp
acl Safe_ports port 443		# https
acl Safe_ports port 70		# gopher
acl Safe_ports port 210		# wais
acl Safe_ports port 1025-65535	# unregistered ports
acl Safe_ports port 280		# http-mgmt
acl Safe_ports port 488		# gss-http
acl Safe_ports port 591		# filemaker
acl Safe_ports port 777		# multiling http
#
acl CONNECT method CONNECT
#
# Gestion des droits d'acces
http_access deny !Safe_ports
http_access deny CONNECT !SSL_ports
http_access allow localhost manager
http_access deny manager
include /etc/squid/conf.d/*
http_access allow localhost
http_access deny all
http_port 3128 intercept
http_port 3129
coredump_dir /var/spool/squid
refresh_pattern ^ftp:		1440	20%	10080
refresh_pattern ^gopher:	1440	0%	1440
refresh_pattern -i (/cgi-bin/|\?) 0	0%	0
refresh_pattern .		0	20%	4320
cache_effective_user proxy
cache_effective_group proxy
url_rewrite_program /usr/bin/squidGuard
`

iptables :
`root@rpiap:/etc/lighttpd/ iptables -L
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
root@rpiap:/etc/lighttpd/ iptables -t nat -L
Chain PREROUTING (policy ACCEPT)
target     prot opt source               destination         
DNAT       tcp  --  anywhere             anywhere             tcp dpt:http to:10.3.141.1:3128
REDIRECT   tcp  --  anywhere             anywhere             tcp dpt:http redir ports 3128

Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         

Chain POSTROUTING (policy ACCEPT)
target     prot opt source               destination         
MASQUERADE  all  --  anywhere             anywhere            
MASQUERADE  all  --  192.168.50.0/24     !192.168.50.0/24     
root@rpiap:/etc/lighttpd/ 
`

Ligthttpd:
`root@rpiap:/etc/lighttpd/ netstat -ntlup | grep :80
tcp        0      0 0.0.0.0:80              0.0.0.0:*               LISTEN      46782/lighttpd      
tcp6       0      0 :::80                   :::*                    LISTEN      46782/lighttpd      
root@rpiap:/etc/lighttpd/ netstat -ntlup | grep :443

root@rpiap:/etc/lighttpd/ grep -A 2 -B 2 deny lighttpd.conf 

index-file.names            = ( "index.php", "index.html" )
url.access-deny             = ( "~", ".inc" )
static-file.exclude-extensions = ( ".php", ".pl", ".fcgi" )

root@rpiap:/etc/lighttpd/ grep -A 2 -B 2 deny ./*/*
`
any ideas ?

**Comment by bmaire28** [comment]
in fact, raspap is only accessible from lan (through hotspot wlan0), not from wan (eth0).

**Reply by bmaire28** [reply]
I've modify my squid.conf and I got another error message:
The following error was encountered while trying to retrieve the URL: [/](http://10.3.141.1/)

    Invalid URL

Some aspect of the requested URL is incorrect.

Some possible problems are:

    Missing or incorrect access protocol (should be "http://" or similar)

    Missing hostname

    Illegal double-escape in the URL-Path

    Illegal character in hostname; underscores are not allowed.

Your cache administrator is [webmaster](mailto:webmaster?subject=CacheErrorInfo%20-%20ERR_INVALID_URL&body=CacheHost%3A%20rpiap%0D%0AErrPage%3A%20ERR_INVALID_URL%0D%0AErr%3A%20%5Bnone%5D%0D%0ATimeStamp%3A%20Fri,%2029%20Dec%202023%2014%3A17%3A03%20GMT%0D%0A%0D%0AClientIP%3A%2010.3.141.50%0D%0A%0D%0AHTTP%20Request%3A%0D%0A%0D%0A%0D%0A).

**Reply by billz** [reply, marked as answer]
It's possible one of the other services you've installed has a web service that could be conflicting with RaspAP.

This isn't usually a problem as RaspAP will happily run alongside another web server. Two methods for doing this are described in this FAQ: https://docs.raspap.com/faq/#webport

**Reply by bmaire28** [reply]
Thank you Billz.

It works perfectly now. 
I'm still having incident on blocked sites like squid-cache.org and other unblocked like play boy put it is another matter.
